charts/actions-runner-controller/templates/auth_proxy_role_binding.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.kube_rbac_proxy.enabled }}
```

Comment:
Woud you mind renaming the key `kube_rbac_proxy` to `metricsProxy` if you're willing to rename `authProxyServiceName` to `metricsServiceName`?